homeassistant/components/holiday/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
That would not be a correct error. We can have multiple instances, just with different country/province.

Reply:
Changed.